content/developer/reference/backend/views.rst

```diff
@@ -1078,21 +1084,11 @@ take the following attributes:
 
 ``templates``
   defines the :ref:`reference/qweb` template ``gantt-popover`` which is used
-  when the user hovers over one of the records in the gantt view.
-
-  The gantt view uses mostly-standard :ref:`javascript qweb
-  <reference/qweb/javascript>` and provides the following context variables:
-
-  .. rst-class:: o-definition-list
-
-  ``widget``
-    the current :js:class:`GanttRow`, can be used to fetch some
-    meta-information. The ``getColor`` method to convert in a color integer is
-    also available directly in the template context without using ``widget``.
+  by the popover that is opened when the user click on a record in the gantt view.
 
-  ``on_create``
-    If specified when clicking the add button on the view, instead of opening a generic dialog, launch a client action.
-    this should hold the xmlid of the action (eg: ``on_create="%(my_module.my_wizard)d"``
+``on_create``
+  If specified when clicking the add button on the view, instead of opening a generic dialog, launch a client action.
+  this should hold the xmlid of the action (eg: ``on_create="%(my_module.my_wizard)d"``
```

Comment:
```suggestion
  If specified, launch a client action when clicking the add button on the view instead of opening a generic dialog.
  This should hold the xmlid of the action (eg: ``on_create="%(my_module.my_wizard)d"``).
```